Phonebook.Frontend/src/app/pages/dashboard/dashboard.component.html

```diff
@@ -81,15 +81,38 @@ <h1 i18n="DashboardComponent|Title of the Bookmarked people section@@DashboardCo
         </mat-form-field>
       </div>
       <!-- This is a workaround for https://github.com/angular/material2/issues/13372 -->
-      <div cdkDropListGroup class="pb-bookmarks-list pb-flex-row">
+      <div cdkDropListGroup class="pb-flex-row">
+        <div *ngIf="layoutSetting === 'view_list'" class="width">
```

Comment:
Please use the enum for comparison something like: (But you would have to add the enum to the `dashboard.component.ts`
```suggestion
        <div *ngIf="layoutSetting === Layout.view_list" class="width">
```

Reply:
I just tried replacing the enum in the enum folder and adding the enum to the dashboard.component.ts. Unfortunately, that didn't work very well. I think the reason for that is that the enum are imported into the App.state. Whenever, I try to change the place from which it is imported from the enum folder to dashboard.component.ts localhost shows up empty.

Reply:
I just found out that the reason why this doesn't work it that if the enum is in the dashboard.component.ts and I then import that into App.state.ts, it creates a circular dependecy since I also import serveral classes from App.state.ts into dashboard.component.ts. Apparently, that doesn't work.